Dockerfile

```diff
@@ -25,10 +25,11 @@ COPY --from=build /messaging/packages/base/package.json packages/base/package.js
 
 COPY --from=build /messaging/package.json package.json
 COPY --from=build /messaging/yarn.lock yarn.lock
-COPY --from=build /messaging/.yarn .yarn
+COPY --from=build /messaging/.yarn/plugins/@yarnpkg/plugin-workspace-tools.cjs .yarn/plugins/@yarnpkg/plugin-workspace-tools.cjs
+COPY --from=build /messaging/.yarn/releases .yarn/releases
```

Comment:
Here I only copied the plugin and yarn bin so since we don't require any other files from the `.yarn` folder. `node_modules` are installed on line 32.

Reply:
Edit: I had to copy all the plugins since yarn expects them all. Probably because the `.yarnrc` file lists them all.